internal/rest/leveldbreceipt_test.go

```diff
@@ -24,7 +24,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/kaleido-io/ethconnect/internal/kvstore"
+	"github.com/hyperledger-firefly/firefly-ethconnect/internal/kvstore"
```

Comment:
I think these should be `hyperledger-labs` not `hyperledger-firefly`

Reply:
🤦🏼